SampleViewer/SampleViewer.pro

```diff
@@ -39,6 +39,10 @@ exists($$PWD/../../../DevBuildCpp.pri) {
   # include the toolkitcpp.pri, which contains all the toolkit resources
   include($$PWD/../../toolkit/uitools/toolkitcpp/toolkitcpp.pri)
 
+  # Include Calcite
+  RESOURCES += $$PWD/../../toolkit/calcite/Calcite/calcite.qrc
+  QML_IMPORT_PATH += $$PWD/../../toolkit/calcite/
```

Comment:
I see below we check for the existence of the .pri. Do we check if the toolkit is even present? I imagine we would want to check that and if it's there we can probably assume it's contents should be there as well 🤷